alpakaConfig.cmake

```diff
@@ -512,18 +674,36 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
     MESSAGE(STATUS ALPAKA_ACC_GPU_CUDA_ENABLED)
 ENDIF()
 
+#------HIP enabled------
+IF(ALPAKA_ACC_GPU_HIP_ENABLE)
+    LIST(APPEND _ALPAKA_COMPILE_DEFINITIONS_PUBLIC "ALPAKA_ACC_GPU_HIP_ENABLED")
+    MESSAGE(STATUS ALPAKA_ACC_GPU_HIP_ENABLED)
+ENDIF()
+#-----------------------
 LIST(APPEND _ALPAKA_COMPILE_DEFINITIONS_PUBLIC "ALPAKA_DEBUG=${ALPAKA_DEBUG}")
 
 IF(ALPAKA_CI)
     LIST(APPEND _ALPAKA_COMPILE_DEFINITIONS_PUBLIC "ALPAKA_CI")
 ENDIF()
 
+#-----------------------
+# workaround to get the flags correctly to the hip compiler
+IF(ALPAKA_ACC_GPU_HIP_ENABLE)
+    SET(hip_defs ${_ALPAKA_COMPILE_DEFINITIONS_PUBLIC})
+    list_add_prefix("-D" hip_defs)
+    LIST(APPEND HIP_HIPCC_FLAGS
```

Comment:
The other thing with the includes from above should most probably also be done here.